assets.json

```diff
@@ -1,235 +1,303 @@
 [
   {
-    "caip-20": "",
+    "caip-19": null,
+    "logo": "/trustwallet/assets/blockchains/cardano/info/logo.png",
```

Comment:
@antoineherzog , this logo path doesn't seem very useful to me.  How do you envision it being used by clients?

Reply:
can we embed the logo.png into the npm maybe? I don't know